doc/releases/migration-guide-4.0.rst

```diff
@@ -156,6 +156,17 @@ Bluetooth Audio
   This needs to be added to all instances of VCP Volume Renderer callback functions defined.
   (:github:`76992`)
 
+* The Unicast Server has a new registration function :code:`bt_bap_unicast_server_register` which
+  takes a :code:`struct bt_bap_unicast_server_register_param` as argument. This allows the Unicast
+  Server to dynamically register Source and Sink ASE count at runtime. The old
+  :code:`CONFIG_BT_ASCS_ASE_SRC_COUNT` and :code:`CONFIG_BT_ASCS_ASE_SNK_COUNT` has been renamed to
+  reflect that they now serve as a compile-time maximum configuration of ASEs to be used.
+  :code:`bt_bap_unicast_server_register` needs to be called once before using the Unicast Server,
+  and more specfically prior to calling :code:`bt_bap_unicast_server_register_cb` for the first
+  time. It does not need to be called again until the new function
+  :code:`bt_bap_unicast_server_unregister` has been called.
+  (:github:`e5fa2`)
```

Comment:
```suggestion
  (:github:`76632`)
```

Reply:
Why?
Actually its says 'b186720ea92d597603b957edb69809b07f2d622f' upstream for this commit

Reply:
It the PR number, not commit SHA :)